dev/Common/TerminalVelocityFeatures-PushNotifications.h

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// THIS FILE IS AUTOMATICALLY GENERATED; DO NOT EDIT IT
+
+// INPUT FILE: dev\common\TerminalVelocityFeatures-DeploymentAPI.xml
+// OPTIONS: -Channel Experimental -Language C++ -Namespace Microsoft.Windows.PushNotifications
```

Comment:
FWIW: Terminal does not check these files in because they're generated by an always-run build step, even in local VS builds. Is it expected that this file will stay in sync when the person holding the repo changes which channel they're building for?

Reply:
>Is it expected

No. But folks generally don't do inner-loop work for specific channels; it's heading for main, then cherrypick if need be. We have other files we checkin a baseline for inner-loop but the build generates updates/overrides; those inner-loop files aren't channel-savvy either. Something we'll need to address longer-term but not a problem today.